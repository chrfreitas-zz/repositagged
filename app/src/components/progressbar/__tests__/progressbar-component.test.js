import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Progressbar from '../index';

describe('Progressbar view', () => {
  it('should match with snapshot', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<Progressbar />);
    expect(result).toMatchSnapshot();
  });
});
