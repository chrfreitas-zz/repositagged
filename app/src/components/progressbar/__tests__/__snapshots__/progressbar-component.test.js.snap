// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progressbar view should match with snapshot 1`] = `
<div
  className="progressbar"
>
  <div
    className="progressbar__icon"
  />
  
</div>
`;
